src/Scriban/Functions/MathFunctions.cs

```diff
@@ -335,5 +337,48 @@ public static object Times(TemplateContext context, SourceSpan span, object valu
         {
             return ScriptBinaryExpression.Evaluate(context, span, ScriptBinaryOperator.Multiply, value, with);
         }
+
+        /// <summary>
+        /// Creates a new UUIDv4
+        /// </summary>
+        /// <returns>The created UUID, ex. 2dc55d50-3f6c-446a-87d0-a5a4eed23269</returns>
+        /// <remarks>
+        /// ```scriban-html
+        /// {{ math.uuid_v4 }}
+        /// ```
+        /// ```html
+        /// 1c0a4aa8-680e-4bd6-95e9-cdbec45ef057
+        /// ```
+        /// </remarks>
+        public static string UuidV4()
```

Comment:
You can expose `uuid` instead of `uuid_v4`. The API is mostly user oriented that don't care whether it is v4 or v1.

Reply:
Ok, changed this to `uuid`.